packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.test.js

```diff
@@ -81,7 +81,7 @@ describe('<SpeedDialAction />', () => {
       const handleClick = spy();
       const wrapper = mount(<SpeedDialAction {...defaultProps} open onClick={handleClick} />);
       const buttonWrapper = wrapper.childAt(0);
-      buttonWrapper.simulate('click');
+      buttonWrapper.props().onClick();
```

Comment:
This seems like a backwards step, what was the reason for this change?

Reply:
Hello, sorry for the delayed response. I went for this approach

```diff
-      buttonWrapper.simulate('click');
+      buttonWrapper.props().onClick();
```

because `simulate` was not updating my `handleClick.callCount` while `onClick()` was.

Reply:
What happens if you use simulate but change the line above to wrapper.find(“button”) ?

Reply:
Then it works like a charm 🎉 🌮 I have reverted my change =) `simulate` works 💯